lms/templates/main.html

```diff
@@ -74,8 +74,7 @@
     window.baseUrl = "${settings.STATIC_URL}";
     (function (require) {
     % if settings.DEBUG is True:
-        ## Using what amounts to a random number in the Development environment for cache-busting
-        var urlArgs = "bust=" + (new Date()).getTime();
```

Comment:
I mistakenly committed this change. I'll revert it. Basically it removes the awkward cache-busting strategy the LMS uses in dev environments in order to never cache any JS.